src/deprecated.jl

```diff
@@ -1325,8 +1325,22 @@ end
 import Base: vcat
 @deprecate vcat(x::Vector{<:AbstractDataFrame}) vcat(x...)
 
-@deprecate showcols(df::AbstractDataFrame, all::Bool=false, values::Bool=true) describe(df, stats = [:eltype, :nmissing, :first, :last])
-@deprecate showcols(io::IO, df::AbstractDataFrame, all::Bool=false, values::Bool=true) show(io, describe(df, stats = [:eltype, :nmissing, :first, :last]), all)
+@deprecate showcols(df::AbstractDataFrame, all::Bool=false, values::Bool=true) describe(df, :eltype, :nmissing, :first, :last)
+@deprecate showcols(io::IO, df::AbstractDataFrame, all::Bool=false, values::Bool=true) show(io, describe(df, :eltype, :nmissing, :first, :last), all)
+function StatsBase.describe(df::AbstractDataFrame; stats=nothing)
+    if stats === nothing 
+        _describe(df, [:mean, :min, :median, 
+                       :max, :nunique, :nmissing, 
+                       :eltype])
+    elseif stats === :all 
```

Comment:
This branch should also print the deprecation AFAICT.

Reply:
fixed.